code/modules/surgery/tools.dm

```diff
@@ -454,3 +454,33 @@
 	attack_verb_simple = list("pumps", "siphons")
 	tool_behaviour = TOOL_BLOODFILTER
 	toolspeed = 1
+
+/obj/item/bag_valve_mask
+	name = "bag valve mask"
+	desc = "When you want someone to breathe less than you want to kiss them."
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "bag valve mask"
+	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
+	custom_materials = list(/datum/material/iron=2500)
+	item_flags = SURGICAL_TOOL
+	w_class = WEIGHT_CLASS_SMALL
+	attack_verb_continuous = list("pumps", "bags")
+	attack_verb_simple = list("pumps", "bags")
+	tool_behaviour = TOOL_AIRBAG
```

Comment:
you dont use these tool behaviours anywhere, why have this?

Reply:
I was following the other tool designs. If it's safe to remove those I can.

Reply:
It's fine as is.